CHAGELOG-template.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+_These releases can contain markup changes and potentially breaking changes.
+A :warning: icon indicates a change that might be a breaking change for your project._
+
+## [Version #]
+<!-- 
+Every release should have an entry.
+Releases should be listed in reverse chronological order, with the newest release listed first.
+-->
+
+[Release date]
+<!-- 
+Include the date the version was released.
+The date should be in the following format: January 1, 2000. -->
+
+[Itemized list of all user-affected changes]
+<!--
+Example format:
+```
+- [Brief statement that summarizes the change] (#[Related PR])
+  - [Optional 1-2 sentence description included in a nested list item.]
+  - [:warning: Summary of breaking or markup updates 
+    with explanation of what action the user must take
+    These items must be flagged with a :warning: icon at the start of the statement.]
+```
+Whenever possible, change items should:
+- Include a brief description of the update that focuses on user benefit
+- Use a consistent set of verbs to begin each statement
+  Examples: "Fixed", "Added", "Improved", "Optimized", "Updated" at the start of the summary helps indicate change type
+- Use plain language and be human-readable
+- Make content scannable by keeping lines short
+- Include a link to the related PR
+- Be categorized under the appropriate section headings, found below
+-->
+
+<!-- Categorize items into the following section headings.
+If it isn't clear how to categorize an item, put it inside the `General` section.
+
+### General
+
+### Performance
+
+### Accessibility
+
+-->
+
+### Dependency updates
+<!-- 
+Share any dependency updates using this example table structure:
+| Name          | Old | New   |
+| :------------ | --- | ----- |
+| concurrently  | ―   | 7.3.0 |
+
+If no updates, write `_No dependency updates_` 
+-->
+
+[ ] vulnerabilities in regular dependencies
+
+Internal only: [ ]moderate, [ ] high vulnerabilities in devDependencies (development dependencies)
+
+Release TGZ SHA-256 hash:[ ]
```

Comment:
Would this be a requirement for every component? We only currently generate for the big USWDS package.